packages/server/graphql/types/User.ts

```diff
@@ -562,11 +562,27 @@ const User: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<any, GQLC
       ) => {
```

Comment:
+1 The `teams` description is: `all the teams the user is on that the viewer can see.`, but we're returning all the teams in the orgs that the user belongs to.

I think it would be worth checking if the `User.teams` field is being used anywhere where the expected values are just the teams that the user is on. 

If that is the case, we should probably create a new field on the User object. If it's not the case, I'd suggest updating the description.